shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/network/helpers/TargetType.kt

```diff
@@ -0,0 +1,23 @@
+package co.nimblehq.blisskmmic.data.network.helpers
+
+import io.ktor.client.request.*
+import io.ktor.http.*
+
+interface TargetType {
+
+    var baseURL: String
+    var path: String
+    var method: HttpMethod
+    var body: Any?
+    var headers: Map<String, String>?
```

Comment:
they should be defined as `val`

```suggestion
    val baseURL: String
    val path: String
    val method: HttpMethod
    val body: Any?
    val headers: Map<String, String>?
```